### PR TITLE
LPS-189726 | Unignore test and modify the test can assert correct download file name

### PR DESCRIPTION
**Pull request by Hijky**
Background:
Unignore test and modify the test can assert correct download file name

Jira ticket:
https://liferay.atlassian.net/browse/LPS-189726

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
The failures are not related to my PR, it's about https://liferay.atlassian.net/browse/LRQA-81653
cc: @magjed4289 @yanan

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase [hunk 1]
```diff
@@ -59,9 +57,7 @@ definition {
 		task ("Then the sample file download successfully") {
 			var sampleFileName = TestCase.getDownloadedTempFileName(fileNamePattern = "objectentry-*.csv");
 
-			ImportExport.assertFileContent(
-				comparisonFileName = "json_objectEntry_export.txt",
-				fileName = ${sampleFileName});
+			ImportExport.assertDownloadedFileName(downloadedFile = ${sampleFileName});
```
Comment (yanan) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase:60]: @Hijky , just one concern that do we need to assert the fileContent?

Comment (Hijky) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase:60]: '`Then the sample file download successfully`', for this task description, I think we need it to assert the download sample file has been downloaded successfully

Comment (magjed4289) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase:60]: from what I see we did check the content earlier, why do we want to modify it now? 🤔

Comment (Hijky) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase:60]: @magjed4289 And I found the [PR](https://github.com/liferay-headless/liferay-portal/pull/1278/files) who created it and I'm not sure if he checked the results for his PR, and when I unignore the test case to run the test, it failed, so I have to modify it

Comment (magjed4289) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase:60]: it looks like we modify the test cause it fails. If it started to fail that means maybe it's a bug?

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**